saleor/payment/utils.py

```diff
@@ -41,7 +39,7 @@
     "currency",
 }
 ALLOWED_GATEWAY_KINDS = {choices[0] for choices in TransactionKind.CHOICES}
-GATEWAYS_META_LABEL = "gateways"
+GATEWAYS_META_NAMESPACE = "gateways"
```

Comment:
btw. Is `gateways` enough meaningfully? It doesn't mean payment_gateway. It will be inside generic meta fields which should contain meaningfully names.

Reply:
Good point, not really part of this PR, but maybe sooner it is changed the better.
Any suggestions regarding best name for this?

Reply:
@maarcingebala  is the best with figuring the better names 🗡 💃